### DISCUSSION TITLE
Use custom driver

### DISCUSSION THREAD

**Discussion by cbolumar** [hassio-addons/app-nut#269] [General]
I have problems with my Salicru 850 home UPS connected to HA OS in Virtualbox using  the standard usbhid-ups driver.  I had the same problems in debian and  built a custom driver that work in debian , how can use my custom driver in HA OS ? In witch path is the addon looking for the driver?

Is posible to change the  driver path in the addon configuration?

**Comment by frenck** [comment]
You can't add custom drivers to this add-on.

If you need that, this add-on is not suitable for you.

../Frenck
